oxipng.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: oxipng
+  version: 9.1.3
+  epoch: 0
+  description: Multithreaded PNG optimizer written in Rust
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - openssf-compiler-options
```

Comment:
we can omit this, this is the default now. ref: https://github.com/orgs/wolfi-dev/discussions/33052